.eslintrc

```diff
@@ -39,7 +39,10 @@
   "globals": {
     "exports": false,
     "module": false,
-    "require": false
+    "require": false,
+    "process": true,
```

Comment:
shouldn't this one be already part of the `node` env? (and maybe some of the new additions as well)